gen-ai/inference/vllm-rayserve-inf2/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# docker build -t ray2.32.0-neuron-vllm .
+FROM rayproject/ray:2.32.0-py311
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+
+RUN sudo apt-get update \
+      && sudo apt-get install -y --no-install-recommends \
+            ca-certificates \
+            autoconf \
+            automake \
+            build-essential \
+            git \
+            gperf \
+            libre2-dev \
+            libssl-dev \
+            libtool \
+            libcurl4-openssl-dev \
+            libb64-dev \
+            libgoogle-perftools-dev \
+            patchelf \
+            python3-dev \
+            python3-pip \
+            python3-setuptools \
+            rapidjson-dev \
+            scons \
+            software-properties-common \
+            pkg-config \
+            unzip \
+            wget \
+            zlib1g-dev \
+            libarchive-dev \
+            libxml2-dev \
+            libnuma-dev \
+            wget \
+            gnupg2 \
+      && sudo rm -rf /var/lib/apt/lists/*
+
+RUN sudo wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB > ./GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB
```

Comment:
These can be grouped into one layer by having only 1 `RUN` command and using `&& \ ` between all of the commands, like: 
```
RUN sudo wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB > ./GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB && \
sudo gpg --no-default-keyring --keyring ./aws_neuron_keyring.gpg --import ./GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB && \
sudo gpg --no-default-keyring --keyring ./aws_neuron_keyring.gpg --export > ./.etc/apt/trusted.gpg.d/aws_neuron.gpg && \
sudo rm ./GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB
```